### PR TITLE
[Staging] Allow url hash to work

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

Allow endpoint to go directly to topic with the topic id as hash on the url.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: takwimu/takwimu_ui/src/components/AnalysisContent/OtherInfoNav.js [hunk 1]
```diff
@@ -29,9 +29,20 @@ const styles = theme => ({
...
     marginBottom: '0.5rem'
+  },
+  navigation: {
+    backgroundColor: 'unset',
+    padding: 0,
+    [theme.breakpoints.up('md')]: {
+      width: '100%'
+    },
+    [theme.breakpoints.up('lg')]: {
+      width: '100%'
```
Comment (kilemensi) [inline takwimu/takwimu_ui/src/components/AnalysisContent/OtherInfoNav.js]: NP: Unnecessary, will use 'md' value.

**Comment by kilemensi** [comment]
> Good start but doesn't work when loaded with url with hash @karimkawambwa

More info: Goes up to profile section but not topic, opens the first topic in the section.

**Comment by karimkawambwa** [comment]
> > Good start but doesn't work when loaded with url with hash @karimkawambwa
> 
> More info: Goes up to profile but not topic, opens the first topic in the profile ie.g. http://localhost:3000/profiles/tanzania#3208f334-975a-45cb-9476-bfedb7b7e1be
> 
> NB: Seems to work in profile sections.

Thanks for the further breakdown.
Its because the initial section does not have a slug.
Should be an easy fix, let me get this in.

File: takwimu/takwimu_ui/src/components/AnalysisContent/OtherInfoNav.js (hunk 1 above)
Comment (karimkawambwa) [reply takwimu/takwimu_ui/src/components/AnalysisContent/OtherInfoNav.js]: Necessarily to override ContentSection root.

**Pull request merged**